### ISSUE TITLE
Can not create validator

### ISSUE THREAD

**Issue by xni** [anoma/anoma#603]
```
root@Ubuntu-2004-focal-64-minimal ~/anoma # anoma client init-validator --alias validators_support --source my-new-acc
Nov 02 00:16:26.079  INFO anoma_apps::cli::context: Chain ID: anoma-feigenbaum-0.ebb9e9f9013
Generating validator account key...
Enter encryption password:
Generating consensus key...
Enter encryption password:
Generating staking reward account key...
Enter encryption password:
Looking-up public key of atest1v4ehgw36gyeyv33cxdqnwv29gce5zwpn8yerwd6yxeq52s2px3pnvv3sx56nsd2xx5urj3fs0527dw from the ledger...
Enter decryption password:
Transaction added to mempool: Response { code: Ok, data: Data([]), log: Log("Process proposal accepted this transaction"), hash: transaction::Hash(092819ECAA59813F439AEEA7440B1E5B5F0264354E36D1BDD3051443F3BD94F9) }
Transaction applied with result: {
  "info": "Transaction is valid. Gas used: 2449012; Changed keys: #atest1v4ehgw36xqmr2d3nx3ryvd2xxgmrq33j8qcns33sxezrgv6zxdzrydjrxveygd2yxumrsdpsf9jc2p/balance/#atest1v4ehgw36gc6yxvpjxccyzvphxycrxw2xxsuyydesxgcnjs3cg9znwv3cxgmnj32yxy6rssf5tcqjm3, #atest1v4ehgw36xqmr2d3nx3ryvd2xxgmrq33j8qcns33sxezrgv6zxdzrydjrxveygd2yxumrsdpsf9jc2p/balance/#atest1v4ehgw36g3zyvs3sxvur2se5g3p5zd33x5uyvvfcg5mrjwzr8qersvjxxdryg32yxpzrw33hw6e55t; VPs result:  Accepted: atest1v4ehgw36gc6yxvpjxccyzvphxycrxw2xxsuyydesxgcnjs3cg9znwv3cxgmnj32yxy6rssf5tcqjm3, atest1v4ehgw36g3zyvs3sxvur2se5g3p5zd33x5uyvvfcg5mrjwzr8qersvjxxdryg32yxpzrw33hw6e55t, atest1v4ehgw36xqmr2d3nx3ryvd2xxgmrq33j8qcns33sxezrgv6zxdzrydjrxveygd2yxumrsdpsf9jc2p;",
  "height": "28001",
  "hash": "906AF7E1D7AE2AA6ECEB2C6E47DEB567D97194BCDF5610D78D48385C16250102",
  "code": "0",
  "gas_used": "2449012",
  "initialized_accounts": []
}
Expected two accounts to be created
Error:
   0: anomac command failed.

Location:
   apps/src/bin/anoma/cli.rs:91
```

**Comment by tzemanovic** [comment]
Thank you for the report! It seems that the result the client received is from a different transaction. We’re looking into it

**Comment by tzemanovic** [comment]
This is a bug in the client's tx response parsing that can occur when there are multiple txs in a block. All the txs results are included in the matched event and we need to find and select the right one

**Comment by tzemanovic** [comment]
this has been fixed and included in v0.3.0 release
